src/__tests__/text-matchers.js

```diff
@@ -194,3 +194,115 @@ cases(
     },
   },
 )
+
+// A good use case for a custom normalizer is stripping
+// out UCC codes such as LRM before matching
```

Comment:
Can we expand out acronyms like this for those who might be unfamiliar with Unicode (UCC -> Unicode Control Character)? 😄

Reply:
Yep, I did expand it out in the user-facing docs, but I'll expand it here too.